runtime-config-linux.md

```diff
@@ -180,7 +180,14 @@ For more information, see [the memory cgroup man page](https://www.kernel.org/do
 
 #### Set oom_score_adj
 
-More information on `oom_score_adj` available [here](https://www.kernel.org/doc/Documentation/filesystems/proc.txt).
+oom_score_adj sets heuristic regarding how the process is evaluated by the kernel during memory pressure.
```

Comment:
“oom_score_adj” should probaby be “**`oom_score_adj`**” or “`oom_score_adj`”.  And maybe reword to:

> **`oom_score_adj`** adjusts the kernel's process evaluation during memory pressure.